src/status_im/contexts/chat/messenger/messages/list/view.cljs

```diff
@@ -409,12 +210,9 @@
         :bounces                           false
         :header                            (when (= (:chat-type chat) constants/private-group-chat-type)
                                              [list-group-chat-header chat])
-        :footer                            [list-footer
-                                            {:theme                  theme
-                                             :chat                   chat
-                                             :window-height          window-height
-                                             :distance-from-list-top distance-from-list-top
-                                             :customization-color    customization-color}]
+        :footer                            [more-messages-loader
+                                            {:chat-id       chat-id
+                                             :window-height window-height}]
         :data                              messages
```

Comment:
I'd really like to check the rendering for the chat screen and improve the implementation to make it more optimal 🙂

As I said in another response, we could file an issue to check this and improve it. Addressing it in this PR would make the review even harder.

CC: @ilmotta

Reply:
Nice suggestion @clauxx, this can work well, but worth measuring first to be sure it's worth it.

> As I said in another response, we could file an issue to check this and improve it. Addressing it in this PR would make the review even harder.

This is the key for us to keep `develop` releasable. PRs with focused scope and with clear requirements for QAs and devs to verify. Performance optimizations are much easier to test in isolation.